Makefile

```diff
@@ -1,5 +1,8 @@
-.PHONY: default
-default: help
+DOCKER_TAG = dev
+GULP = node_modules/.bin/gulp
+PYTHON_VERSION = 3.6.8
+PYTHON = ${PYENV_ROOT}/versions/${PYTHON_VERSION}
+TOX = ${PYTHON}/bin/tox
```

Comment:
Variable definitions need to be at the top of the file, not at the bottom, otherwise variable references evaluate to empty strings when used in certain places (targets, prerequisites) though not when used in others (recipes). I've moved the existing variables up here along with the new ones in order to keep all the variable definitions together in one place.